### PR TITLE
README: highlight quick-start and update the contact-us chapter

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: shirly <AndreMouche@126.com>

### What problem does this PR solve?

highlight quick-start and update the contact-us chapter

### What is changed and how it works?

we updated the document of readme

### Related changes

no

### Check List 

- No code

Side effects

readme

### Release note 
- No release note

### CODE REVIEW

**Comment by AndreMouche** [comment]
@winkyao PTAL

**Comment by AndreMouche** [comment]
@shenli @winkyao PTAL

**Comment by winkyao** [comment]
What about moving our contact information to the bottom?

File: README.md [hunk 1]
```diff
@@ -49,6 +49,23 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [official TiDB blog](https://www.pingcap.com/blog/).
 
+
+## Quick start
+
+### To start using TiDB
```
Comment (shenli) [inline README.md:55]: Use TiDB

File: README.md [hunk 2]
```diff
@@ -49,6 +49,23 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
...
+
+Read the [Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/ ), which includes deployment methods using TiUP, Ansible, Docker, and Kubernetes.
+
+### To start developing TiDB
```
Comment (shenli) [inline README.md:59]: Contribute to TiDB

File: README.md (hunk 1 above)
Comment (AndreMouche) [reply README.md:55]: It was referred from k8s https://github.com/kubernetes/kubernetes#to-start-using-kubernetes

File: README.md (hunk 2 above)
Comment (AndreMouche) [reply README.md:59]: Ditto, referred from k8s https://github.com/kubernetes/kubernetes#to-start-developing-kubernetes

File: README.md [hunk 3]
```diff
@@ -49,6 +49,23 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
...
+
+The [community repository](https://github.com/pingcap/community) hosts all information about building TiDB from sourse, how to contribute code and documentation, how to join special interest group, who to contact about what, etc.
```
Comment (dcalvin) [inline README.md:61]:
```suggestion
The [community repository](https://github.com/pingcap/community) hosts all information about the TiDB community, including how to contribute to TiDB, how TiDB community is governed,  how special interest groups are organized, etc.
```

**Pull request merged**